projects/plugins/jetpack/extensions/shared/external-media/media-service/index.ts

```diff
@@ -196,6 +196,10 @@ const isMediaSourceConnected = async ( source: MediaSource ) =>
  * @returns {boolean} True if the inserter is opened false otherwise.
  */
 const isInserterOpened = (): boolean =>
+	// @to-do: remove exception when Jetpack requires WordPress 6.5.
+	// eslint-disable-next-line
+	// @ts-ignore
+	select( 'core/editor' )?.isInserterOpened?.() ||
```

Comment:
This doesn't work, I still get a `Uncaught TypeError: o.select(...).isInserterOpened is not a function`. I don't understand why. What would be the best approach here?